### ISSUE TITLE
Change the way how tests are run for single packages

### ISSUE THREAD

**Issue by jodator** [ckeditor/ckeditor5#7139]
The [circular dependency of `ckeditor5-paragraph` and `ckeditor5-heading`](https://github.com/ckeditor/ckeditor5-dev/issues/470) showed that we shouldn't be running tests as we do now. The problem is that we run them differently then we do on development environment.

Current workflow:
1. Travis checkouts github repository
2. We bootstrap testing environment:
  - `yarn add ckeditor5-dev-tests`
  - `yarn add mgit` (second yarn add)
  - create `mgit.json` with immediate packages dependencies
  - run `mgit sync` which add all `@ckeditor` dependencies into `./packages` dir
  - run `yarn install`
3. We run test in main package

This worked well with `npm` + `lerna` but fails with `yarn` workspaces. Yarn doesn't symlink main package to the `node_modules` dir and thus we have duplicated dependencies when one of packages requires package that is being tested. (circular deps - mostly in the devDependencies).

After short F2F talk with @Reinmar we think that we should run our testing environment for single packages the same way as we run for all packages and as we do in dev environment.

In short we should create a workspace in which current package is located (as in `ckeditor5` with all it's dependencies) and then run test only for this package.

ps.: Also we should merge `yarn add` calls to a minimum as that doubled `yarn add` isn't necessary and we could merge them into one step.
